src/extension.ts

```diff
@@ -135,6 +135,14 @@ function toggleBazelClasspathSyncStatus(doc: TextDocument){
 }
 
 function toggleBazelProjectSyncStatus(doc: TextDocument){
+	window.showInformationMessage('Project View Updated', 'Sync Project View', 'Sync Directories Only')
```

Comment:
For consistency with the status bar item
```suggestion
	window.showInformationMessage('Project View Updated', 'Sync bazel project view', 'Sync Directories Only')
```

Reply:
Alternatively, the status bar item text could be updated to 'Sync Project View'